apps/etterlatte-behandling/src/main/kotlin/behandling/migrering/MigreringService.kt

```diff
@@ -165,3 +173,8 @@ class MigreringService(
         behandlingService.avbrytBehandling(behandlingId, brukerTokenInfo)
     }
 }
+
+class FinnesLoependeEllerIverksattBehandlingForFnr : Exception()
+
+// TODO Er det en bedre måte å gjøre dette på?
+fun Behandling.erAutomatiskGjenoppretting() = kommerBarnetTilgode?.kilde?.type == Grunnlagsopplysning.Gjenny.create().type
```

Comment:
Kjens litt rart å gå via kommerBarnetTilgode her, men ser ut som du kan gå rett på this.kilde?

Reply:
Ja enig at det er litt hacky. Men kilde på behandlingsnvivå er vedtaksløsning og hvis den er Gjenny så skiller ikke det mellom automatisk gjenoppretting og alle andre behandlinger i Gjenny.

Ellers så er jeg egentlig litt usikker på om gjenoppretting skal ha Gjenny eller ikke. På en side tenker jeg at det skal være Gjenny siden det ikke er en migrering men en gjenoppretting som skjer i Gjenny. Men på en annen side så er jo alle beslutninger og og grunnlag vi bruker gjort i Pesys. 🤔

Reply:
Men det er kanskje så farlig at det feltet er 100% presist? Det er vel mest et felt som er til hjelp for oss?